src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { Link } from 'react-router-dom';
+import { PATHS } from '../../utils/const';
+import { Person } from '../../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  name: string | null;
+};
+
+export const PersonLink: React.FC<Props> = ({ person, name }) => {
+  return (
+    <Link
+      className={classNames('', {
+        'has-text-danger': person.sex === 'f',
```

Comment:
```suggestion
        'has-text-danger': person.sex === 'f',
```
better create enum for 'f' and 'm'